client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+        "confirmDeleteSpot": "Soll die Markierung \"{{spot}}\" wirklich gelöscht werden?",
+        "confirmDeleteZone": "Soll die Zone \"{{zone}}\" wirklich gelöscht werden?",
+        "zonesPageBack": "Zonen",
+        "zoneEditTitle": "Bearbeiten von Zone: {{zone}}",
+        "tooManySubzones": "Es können nicht mehr al 5 Unterzonen in einer Zone gesetzt werden.",
+        "tooManyMarkers": "Es können keine weitere Verbotsmarker gesetzt werden!",
+        "zonesSavedOK": "Zone erfolgreich gespeichert!",
+        "spotsPageBack": "Markierung",
+        "spotEditTitle": "Bearbeiten von Markierung: {{spot}}",
+        "spotsSavedOK": "Markierung erfolgreich gespeichert!",
+        "forbiddenMarkersPageBack": "Verbotsmarkierungen",
+        "forbiddenMarkersTitle": "Markierungen bearbeiten",
+        "forbiddenMarkersSavedOK": "Verbotsmarkierungen erfolgreich gespeichert!"
+    },
+    "control": {
+        "startButton": "Manuelle Kontrolle starten",
+        "stopButton": "Manuelle Kontrolle beenden"
+    },
+    "settings": {
+        "title": "Einstellungen",
+        "infoTitle": "Informationen",
+        "infoDesc": "Geräteinformationen anzeigen",
+        "timersTitle": "Zeitsteuerung",
+        "timersDesc": "Einstellen der Zeitabhängigen Steuerung",
+        "carpetModeTitle": "Teppichmodus",
+        "carpetModeDesc": "Teppichmodus einstellen",
+        "persistentDataTitle": "Dauerhafte Daten",
+        "persistentDataDesc": "Einstellen des Labormodus um virtuelle Wände, etc. zu aktivieren.",
+        "consumablesTitle": "Verbrauchsmaterialien",
+        "consumablesDesc": "Ansehen/Zurücksetzten von Verbrauchszählern",
+        "cleaningHistoryTitle": "Reinigungshistorie",
+        "cleaningHistoryDesc": "Ansehen der Reinigungshistorie",
+        "wifiTitle": "W-LAN",
+        "wifiDesc": "Ansehen/Einstellen der W-LAN Details und Einstellungen",
+        "soundAndVoiceTitle": "Ton & Stimme",
+        "soundAndVoiceDesc": "Ändern der Lautsärke und die Sprache des Roboters",
+        "webInterfaceTitle": "Web Interface",
+        "webInterfaceDesc": "Ändern der Web Interface Parameter",
+        "accessControlTitle": "Zugangskontrolle",
+        "accessControlDesc": "Einstellen von SSH Schlüsseln und HTTP Authentifizierung",
+        "loading": "Laden...",
+        "accessControl": {
+            "pageTitle": "Zugangskontrolleinstellungen",
+            "httpAuth": "HTTP Authentifizierungs Einstellungen",
+            "username": "Benutzername",
+            "password": "Passwort",
+            "passwordRepeat": "Passwort (wiederholen)",
+            "saveCredentials": "Eingaben speichern",
+            "sshKeys": "SSH Schlüssel Einstellungen",
+            "sshRSA": "ssh-rsa ...",
+            "saveKeys": "Speichere SSH Schlüssel",
+            "confirmDisableKeys": "\"confirm\" eingeben. Schliess Dich nicht aus!",
+            "disableKeys": "Dauerhaft das hochladen von SSH Schlüssel abschalten",
```

Comment:
Even the english text almost fills out the line on my phone, this would be too long. Shorter alternative:
"disableKeys": "Schlüsselupload dauerhaft deaktivieren",